/**:
  ros__parameters:
    blockage_ratio_threshold: 0.8
    blockage_count_threshold: 50
    blockage_buffering_frames: 2
    blockage_buffering_interval: 1
    enable_dust_diag: false
    publish_debug_image: true
    dust_ratio_threshold: 0.2
    dust_count_threshold: 10
    dust_kernel_size: 2
    dust_buffering_frames: 10
    dust_buffering_interval: 1
    blockage_kernel: 10
    max_distance_range: 200.0
    horizontal_resolution: 0.4
    angle_range: [0.0, 360.0]
    vertical_bins: 128
    is_channel_order_top2down: true
    horizontal_ring_id: 40
    enable_direct_mask_input: true
